README.md

```diff
@@ -113,3 +122,11 @@ A git extension for managing dependencies that come from git repositories
 
 * `-of` or `--os-filter`: filter dependencies by OS type. Default value is current OS type.
   * **commands** : `update`
+
+
+* `-o` or `--override [filename]`: Override dependencies from the override file `filename`.
+  * **commands** : `update`
+
+
+* `-oa` or `--override-all`: Enforce complete override. Fail if a dependency is not overriden by the override file.
+  * **commands** : `update --override`
```

Comment:
Do I need to specify --override when I want to use --override-all, or is it implied?